com.cubrid.cubridmigration.core.testfragment/src/com/cubrid/cubridmigration/mariadb/MariaDB2CUBRIDMigParasTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2009 Search Solution Corporation. All rights reserved by Search Solution. 
```

Comment:
If we create files, then we use copyright in license statements as:

` * Copyright (c) 2016 CUBRID Corporation.`

If we modified existing files, then we use copyright in license statements as:

```
 * Copyright (C) 2008 Search Solution Corporation.
 * Copyright (c) 2016 CUBRID Corporation.
```

Reply:
Thank you for comment. I would like to confirm:
- create new file:
```
/*
 * Copyright (c) 2016 CUBRID Corporation. All rights reserved by Search Solution. 
```
 
 -  modify file.
```
/*
 * Copyright (C) 2008 Search Solution Corporation.
 * Copyright (C) 2009 Search Solution Corporation. All rights reserved by Search Solution. 
```
Am I right?

Reply:
* when you create new file, copy existing license statement. And replace following line into new one

ASIS

`* Copyright (C) 2009 Search Solution Corporation. All rights reserved by Search Solution.`

TOBE

`* Copyright (c) 2016 CUBRID Corporation.`

-----

* when you **modify** existing file, Replace following line into new one

ASIS

`Copyright (C) 2009 Search Solution Corporation. All rights reserved by Search Solution.`

**TOBE**

```
* Copyright (C) 2008 Search Solution Corporation.
* Copyright (c) 2016 CUBRID Corporation.
```